### ISSUE TITLE
plug-up.com $6 USD, fido U2F

### ISSUE THREAD

**Issue by RiftDraft** [shield-9/u2f-login#6]
Please consider extending support to six dollar u2f security key from plug-up.com available via amazon platform albeit not via amazon prime.

This would be both a suitable backup key, or a primary key.

Please do not rely upon google's privacy raping "free" services as without privacy there can be neither security nor liberty.

**Comment by shield-9** [comment]
@RiftDraft, do you mean [this product](http://www.amazon.com/gp/product/B00OGPO3ZS/)?

**Comment by RiftDraft** [comment]
Yes, @shield-9.   Amazon ASIN: B00OGPO3ZS ;  text ASIN avoids some concern over affiliate url infliction.

Manufacturer propaganda:

http://sk.plug-up.com/

**Comment by shield-9** [comment]
According to Amazon.com, this product could not be shipped to my region, Japan.
Unfortunately, I think it hard to add support for this security key...

**Comment by DefyMiners** [comment]
Store.kliidje.com allows me to set japan as shipping country. I had no address to test but if problematic I could buy you one with the hope that OCRA would be possible if offered in plug-up firmware.

Plug-up.com . buy. Buysecuritykey.com .buy now. Buy on plug-up shop. Store.kliidge.com 

I shall ask amazon if I can dropship to japan.

**Comment by piotron** [comment]
@shield-9  I think you can order it from amazon.fr and amazon.de I've had similar problem.

**Comment by shield-9** [comment]
@piotron,thank you!
I was able to order one from amazon.fr. It will arrive within 2~3 weeks.

(amazon.de seems to be not able to ship to Japan.)

**Comment by doherty** [comment]
If the device talks the U2F protocol to this plugin, then what's to support? It should Just Work.

**Comment by shield-9** [comment]
@doherty, yes, this device has "FIDO U2F Ready", but it was put on the market before publication of FIDO 1.0. So, I'd like to test.

**Comment by piotron** [comment]
I've tested it with yubico implementation, will check it with your in weekend.

**Comment by minisu** [comment]
> Please consider extending support to six dollar u2f security key from plug-up.com available via amazon platform albeit not via amazon prime.

This plugin uses Yubico's U2F library, which supports the U2F standard and is tested with all U2F devices on the market. I just confirmed that this plugin works fine with Plug-up's U2F device. 

> Please do not rely upon google's privacy raping "free" services as without privacy there can be neither security nor liberty.

Who is relying on Google's free services? How is this relevant to this plugin or U2F?

**Comment by Nibbels** [comment]
I have the plug-up Security Key.
With that I just tried to use it with PHP 5.6, Mysql 5.5, Wordpress 4.11, Crome 40.0.2214.115 m and an HTTPS-connection.

I activated the Plugin, fixed the error with the missing Server, went to 'Your Security Key' and klicked on [Register].
It is waiting for an Security Key and doing nothing.
http://downfight.de/grafiken/dimi/fe081632392251617e5af8826a04d7a7.png

This other 'Email-Token' feature of the plugin works.

[I can easily log in with Google-Accounts. I testet two Sticks of the same Type]
